build/lib/EpikCord/managers/__init__.py

```diff
@@ -0,0 +1,15 @@
+"""
+Some parts of the code is done by discord.py and their amazing team of contributors
```

Comment:
But none of their contributors helped to this? This is a new feature we added. Unless we need to add his license to every part of code which may not be theirs?